airbyte-integrations/connectors/source-stripe/erd/discovered_catalog.json

```diff
@@ -3966,6 +3966,49 @@
       "default_cursor_field": ["created"],
       "source_defined_primary_key": [["id"]]
     },
+    {
+      "name": "payout_balance_transactions",
+      "json_schema": {
+        "type": ["null", "object"],
+        "properties": {
+          "fee": { "type": ["null", "integer"] },
+          "currency": { "type": ["null", "string"] },
+          "source": { "type": ["null", "string"] },
+          "fee_details": {
+            "type": ["null", "array"],
+            "items": {
+              "properties": {
+                "application": { "type": ["null", "string"] },
+                "type": { "type": ["null", "string"] },
+                "description": { "type": ["null", "string"] },
+                "amount": { "type": ["null", "integer"] },
+                "currency": { "type": ["null", "string"] }
+              },
+              "type": ["null", "object"]
+            }
+          },
+          "available_on": { "type": ["null", "integer"] },
+          "status": { "type": ["null", "string"] },
+          "description": { "type": ["null", "string"] },
+          "net": { "type": ["null", "integer"] },
+          "exchange_rate": { "type": ["null", "number"] },
+          "type": { "type": ["null", "string"] },
+          "sourced_transfers": { "items": {}, "type": ["null", "array"] },
+          "id": { "type": ["null", "string"] },
+          "object": { "type": ["null", "string"] },
+          "created": { "type": ["null", "integer"] },
+          "updated": { "type": ["null", "integer"] },
+          "amount": { "type": ["null", "integer"] },
+          "reporting_category": { "type": ["null", "string"] },
+          "payout": { "type": ["null", "string"] }
+        }
+      },
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
+      "default_cursor_field": ["updated"],
+      "source_defined_primary_key": [["payout"],["id"]],
+      "user_defined_primary_key": [["payout"],["id"]]
```

Comment:
Not sure if I need this or just the source_defined_primary_key - I saw conflicting documentation on if airbyte did or did now support composite primary keys. In theory ID would be enough, but I don't know 100% if a balance transaction could be linked to multiple payouts.

Reply:
composite PKs made of two+ columns generally don't work very well in Airbyte, and I remember we're trying to avoid them.

Reply:
Maybe in this case we can create an artificial PK combining `{id}_{payout}` @maxi297 do you have any thoughts here?

Reply:
We can probably do that, but can @natikgadzhi expand on this comment? I thought Airbyte wasn't dropping support for nested PKs but not composite ones. I would like to clarify that because we have a lot of sources that have composite PKs like source-google-analytics-data-api or source-google-search-console